model/sys/vmem_utils.sail

```diff
@@ -172,6 +172,8 @@ function vmem_read_addr(vaddr, offset, width, acc, aq, rl, res) = {
   Ok(data)
 }
 
+termination_measure vmem_read_addr repeat n
```

Comment:
It would be helpful to explain the `repeat` construct here, and refer to it for the `vmem_write_addr` case below.

Reply:
I honestly don't know what this does precisely.

It's not very documented... according to the syntax documentation in the SAIL repo this shouldn't even be valid syntax...
<img width="1182" height="108" alt="image" src="https://github.com/user-attachments/assets/b4ef5895-ed73-4828-82dd-1be0cea9317a" />

@Alasdair I think maybe you added this, could you explain to me what this variant of `termination_measure` is?

Reply:
Alright I reverted it!